package.json

```diff
@@ -27,6 +27,8 @@
     "@babel/preset-react": "7.0.0-beta.44",
     "@babel/preset-stage-2": "7.0.0-beta.44",
     "@babel/runtime": "7.0.0-beta.44",
+    "@wordpress/data": "1.0.0",
+    "ajv": "6.5.2",
```

Comment:
why the ajv dep?

Reply:
I was running into this during installation:

`npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.`

Haven't checked if it's really necessary yet.

Reply:
ah ok. i think that's a npm issue. we see it on master too.

Reply:
https://github.com/webpack-contrib/schema-utils/issues/23#issuecomment-367322144

Reply:
Thanks for the additional context! As expected, removing it didn't cause any issues on this branch.